http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3036,6 +3036,10 @@ components:
         defaultClientId:
           type: string
           description: The default Google Client ID.
+        timeoutInMinutes:
+          type: integer
+          description: Custom image deployment time out in minutes.
+            If nothing id provided, a default of 10 minutes will be used.
```

Comment:
one caveat of this is there’s this defines a timeout for the entire create runtime flow…so if user’s input is more than 30 min, it won’t work…I don’t think we need to do anything intelligently update this overall timeout yet..but maybe mention that currently, this timeout can't be more than 30 min? if in the future, user requests to increase this overall timeout, we can do that

Reply:
Ohhh, I was not aware of that 30 minutes timeout! I will definitely update the documentation to reflect that, thanks! @n8landolt it should likely be reflected in the UI (e.g. a validation that makes sure you are getting an integer between 1 and 30)

Reply:
@Qi77Qi I also remember that you would have preferred to name this variable differently, would you like me to change it as well?

Reply:
Maybe let's have a mini huddle with @n8landolt today and decide on the naming for both the ui and leo?

Reply:
just corrected my last comment (think I had an extra n't)...

Reply:
Ahah indeed I get it now! `timeout` on the UI side, but `checkToolsInterruptAfter` in Leo past the endpoint :) FYI @n8landolt